app/models/custom_field.rb

```diff
@@ -150,6 +150,7 @@ def value_types
       when 'range' then [:rating, :text]
       when 'boolean' then [:boolean]
       when 'breakeven_data' then [:breakeven_data]
+      when 'business_canvas' then [:business_canvas_data]
```

Comment:
Oh but then I see you've added an extra `_data` in the business_canvas case. That seems kind of confusing since none of the others have that, even the other complex type (`breakeven_data`). Can you remove?